math/src/elliptic_curve/short_weierstrass/curves/bls12_381/curve.rs

```diff
@@ -1,13 +1,25 @@
 use super::field_extension::{BLS12381PrimeField, Degree2ExtensionField};
+use crate::cyclic_group::IsGroup;
 use crate::elliptic_curve::short_weierstrass::point::ShortWeierstrassProjectivePoint;
 use crate::elliptic_curve::traits::IsEllipticCurve;
+use crate::unsigned_integer::element::*;
 use crate::{
     elliptic_curve::short_weierstrass::traits::IsShortWeierstrass, field::element::FieldElement,
 };
 
 pub type BLS12381FieldElement = FieldElement<BLS12381PrimeField>;
 pub type BLS12381TwistCurveFieldElement = FieldElement<Degree2ExtensionField>;
 
+/// Cube Root of Unity in G1.
```

Comment:
Could you make this comment more clear?
```suggestion
/// Cube root of unity in the base field of BLS 12 381.
```